libraries/usfm_tools/verifyUSFM.py

```diff
@@ -593,28 +594,37 @@ def takeUnknown(state, token):
     value = token.getValue()
     if (value == 'v') or (value == 'c'):
         return  # skip malformed chapter and verses - will be caught later
-    report_error( state.reference + " - Unknown Token: '\\" + value + "'")
+    elif value == 'p':
+        report_error( state.reference + " - Orphan paragraph marker follows")
+    else:
+        report_error( state.reference + " - Unknown Token: '\\" + value + "'")
 
-# Returns True if token is the start of a footnote - note that verse can contain footnote for more reasons than just
-#       does not appear in some manuscripts.
-def isFootnoted(token):
-    state = State()
-    footnoted = token.isFS()
-    # and state.reference in { 'MAT 17:21', 'MAT 18:11', 'MAT 23:14', 'MRK 7:16', 'MRK 9:44', 'MRK 9:46', 'MRK 11:26', 'MRK 15:28', 'MRK 16:9', 'MRK 16:12', 'MRK 16:14', 'MRK 16:17', 'MRK 16:19', 'LUK 17:36', 'LUK 23:17', 'JHN 5:4', 'JHN 7:53', 'JHN 8:1', 'JHN 8:4', 'JHN 8:7', 'JHN 8:9', 'ACT 8:37', 'ACT 15:34', 'ACT 24:7', 'ACT 28:29', 'ROM 16:24' }
-    if footnoted:
```

Comment:
@lversaw I think we've lost the necessary exclusions for verses that only have a footnote on purpose.  For example, [Matthew 18:11](http://dev-door43.org.s3-website-us-west-2.amazonaws.com/u/Door43/en_ulb/4c81c0aaf7/41-MAT.html#fn-040-018-011-1) shouldn't have any main body text, only a footnote.  I think the verse list above is probably the list of verses that are OK even if they don't have text.  Compare the warnings that now show up on http://dev-door43.org.s3-website-us-west-2.amazonaws.com/u/Door43/en_ulb/4c81c0aaf7 .